test/unit/test_ranges.py

```diff
@@ -46,10 +46,8 @@
 
 
 class TestRangeHeaders(unittest.TestCase):
-    """
-    Test outgoing range headers to sproxyd from the object server on a partial
-    get of an object. See
-    https://github.com/scality/ScalitySproxydSwift/issues/121 for more details.
+    """Test outgoing range headers to sproxyd from the object server on a partial
+    get of an object. See ScalitySproxydSwift issue #121 for more details.
```

Comment:
I'd keep the URL to the issue somewhere in here. Much easier to browse to and such. Maybe as a reference in a second paragraph, or use ReST-like formatting?